src/cc/bpf_module.cc

```diff
@@ -295,7 +295,7 @@ unique_ptr<ExecutionEngine> BPFModule::finalize_rw(unique_ptr<Module> m) {
 
 // load an entire c file as a module
 int BPFModule::load_cfile(const string &file, bool in_memory) {
-  clang_loader_ = make_unique<ClangLoader>(&*ctx_);
+  clang_loader_ = make_unique<ClangLoader>(&*ctx_, flags_);
```

Comment:
flags is the debug=?? field of the BPF __init__ constructor. Currently there are users of 0x1, 0x2, and 0x4 bits, but not necessarily in all classes that flags_ is passed is it used.